java/src/main/java/lesson04/solution/RequestBuilderCarrier.java

```diff
@@ -0,0 +1,24 @@
+package lesson04.solution;
+
+import okhttp3.Request;
+
+import java.util.Iterator;
+import java.util.Map;
+
+public class RequestBuilderCarrier implements io.opentracing.propagation.TextMap {
```

Comment:
we keep repeating this class everywhere - any reason not to move this to lib.Tracing?

Reply:
it doesn't even need to be a class, but a static method `lib.Tracing.requestBuilderCarrier()`